web/modules/custom/mymodule/mymodule.routing.yml

```diff
@@ -0,0 +1,23 @@
+mymodule.simplecontent:
+  path: '/mycustompath'
```

Comment:
Please provide proper route name.

Reply:
Should it be something like "/modulename/the page name" ....